### PR TITLE
Add CHANGELOG entries for 11.0.5

### PR DESCRIPTION
**Pull request by onurctirtir**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,26 @@
...
+* Avoids possible information leakage about existing users
+
+* Allows using `WITH HOLD` cursors with parameters
+
+* Fixes a bug that could cause failures in `INSERT INTO .. SELECT`
+
+* Fixes a bug that prevents pushing down `IN` expressions when using columnar
+  custom scan
+
+* Fixes a concurrency bug between creating co-located distributed table and
```
Comment (onurctirtir) [inline CHANGELOG.md:12]:
```suggestion
* Fixes a concurrency bug between creating co-located distributed table and

* Fixes a crash that can happen due to catalog read in shmem_exit
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,28 @@
...
+
+* Fixes a crash that can happen due to catalog read in shmem_exit
+  shard moves
+
+* Fixes an unexpected error caused by constraints when moving shards
+
+* Fixes an unexpected error for foreign tables when upgrading Postgres
+
+* Prevents adding local table into metadata if there is a view with circular
+  dependencies on it
```
Comment (onurctirtir) [inline CHANGELOG.md:22]: made it little more user friendly

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,28 @@
+### citus v11.0.5 (August 1, 2022) ###
+
+* Avoids possible information leakage about existing users
+
+* Allows using `WITH HOLD` cursors with parameters
+
+* Fixes a bug that could cause failures in `INSERT INTO .. SELECT`
+
+* Fixes a bug that prevents pushing down `IN` expressions when using columnar
+  custom scan
```
Comment (onurctirtir) [inline CHANGELOG.md:10]: made up this for https://github.com/citusdata/citus/pull/6030 as it didn't have a DESCRIPTION line

File: CHANGELOG.md (hunk 1 above)
Comment (hanefi) [reply CHANGELOG.md:12]: This suggestion is still missing some parts I guess. The first entry ends with the word `and`

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,28 @@
...
+
+* Fixes a concurrency bug between creating co-located distributed table and
+
+* Fixes a crash that can happen due to catalog read in shmem_exit
+  shard moves
```
Comment (hanefi) [inline CHANGELOG.md:15]:
```suggestion
* Fixes a concurrency bug between creating co-located distributed table and
  shard moves

* Fixes a crash that can happen due to catalog read in shmem_exit
```

File: CHANGELOG.md (hunk 1 above)
Comment (hanefi) [reply CHANGELOG.md:12]: See https://github.com/citusdata/citus/pull/6108/files#r934247807

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,28 @@
...
+* Avoids possible information leakage about existing users
+
+* Allows using `WITH HOLD` cursors with parameters
+
+* Fixes a bug that could cause failures in `INSERT INTO .. SELECT`
+
+* Fixes a bug that prevents pushing down `IN` expressions when using columnar
+  custom scan
+
+* Fixes a concurrency bug between creating co-located distributed table and
```
Comment (onurctirtir) [inline CHANGELOG.md:12]:
```suggestion
* Fixes a concurrency bug between creating a co-located distributed table and
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,28 @@
...
+
+* Fixes a bug that could cause failures in `INSERT INTO .. SELECT`
+
+* Fixes a bug that prevents pushing down `IN` expressions when using columnar
+  custom scan
+
+* Fixes a concurrency bug between creating a co-located distributed table and
+  shard moves
+
+* Fixes a crash that can happen due to catalog read in shmem_exit
```
Comment (hanefi) [inline CHANGELOG.md:15]:
```suggestion
* Fixes a crash that can happen due to catalog read in `shmem_exit`
```

I am not sure if a user would know about `shmem_exit`. Should we paraphrase this one maybe?

**Review by hanefi — APPROVED** [review]
only one comment

**Comment by onderkalaci** [comment]
please do not forget to squash the commits, it seems unusual to have 5 commits updating the same changelog :)

**Comment by onurctirtir** [comment]
> please do not forget to squash the commits, it seems unusual to have 5 commits updating the same changelog :) 

Sure, and will merge the pr as soon as I'm back

File: CHANGELOG.md (hunk 6 above)
Comment (onurctirtir) [reply CHANGELOG.md:15]: It's a bit hard to make this more user-friendly than it is, so preferring to leave this as is

**Pull request merged**